docs/guide/time.rst

```diff
@@ -16,29 +16,37 @@ SunPy builds upon its functionality.
 
 Solar data is associated with a number of different time formats. SunPy provides a simple
 parsing function which can deal with most every format that a user may encounter. Called
-`sunpy.time.parse_time()`, this function takes a string as input and returns a datetime object.
+`sunpy.time.parse_time()`, this function takes a string as input and returns a `~astropy.time.Time` object.
+
 Here are few examples of formats which `sunpy.time.parse_time()` accepts::
 
     >>> from sunpy.time import parse_time
     >>> parse_time('2007-05-04T21:08:12')
-    datetime.datetime(2007, 5, 4, 21, 8, 12)
+    <Time object: scale='utc' format='isot' value=2007-05-04T21:08:12.000>
     >>> parse_time('2007/05/04T21:08:12')
-    datetime.datetime(2007, 5, 4, 21, 8, 12)
+    <Time object: scale='utc' format='isot' value=2007-05-04T21:08:12.000>
     >>> parse_time('20070504T210812')
-    datetime.datetime(2007, 5, 4, 21, 8, 12)
+    <Time object: scale='utc' format='isot' value=2007-05-04T21:08:12.000>
     >>> parse_time('2007-May-04 21:08:12')
-    datetime.datetime(2007, 5, 4, 21, 8, 12)
+    <Time object: scale='utc' format='isot' value=2007-05-04T21:08:12.000>
     >>> parse_time('20070504_210812')
-    datetime.datetime(2007, 5, 4, 21, 8, 12)
+    <Time object: scale='utc' format='isot' value=2007-05-04T21:08:12.000>
 
-Each of the above returns the same datetime object ``datetime.datetime(2007,
-5, 4, 21, 8, 12)``. One of the most standard time formats used in solar
-physics is the number of seconds since 1979 January 01. The parse_time
+Each of the above returns the same `~astropy.time.Time` object ``<Time object: scale='utc' format='isot' value=2007-05-04T21:08:12.000>``. One of the most standard time formats used in solar
```

Comment:
`utime` is exactly  the opposite of "the most standard time format" :stuck_out_tongue: